File: 编译文件/源码/fizzbuzz.ts
//console.log("helloworld")
//var i :number = 1;
for(i= 1;i<100+1;i++) {
  if(i%3== 0 && i%5== 0){
    console.log("FizzBuzz");
  }else if(i%3!= 0 && i%5== 0){
    console.log("Fizz");
  }else if(i%3== 0 && i%5!= 0){
    console.log("Buzz");
  }else{
    console.log(i);
  }
}

File: 编译文件/源码/fizzbuzz.rs
fn main(){
  //println!("helloworld");
  println!("{:#?}",
    (0..100).map(|n :usize| 
    match n.pow(4) % 15 {
      1 => n.to_string(),
      6 => "Fizz".to_string(),
      10 => "Buzz".to_string(),
      0 => "FizzBuzz".to_string(),
      _ => unreachable!()
    }).collect::<Vec<String>>());
}